mathsys/dev/core/data/_infinite.rs

```
@@ -0,0 +1,41 @@
+//^ INFINITE
+//^
+
+//> INFINITE -> STRUCT
+#[derive(Clone)]
+pub struct _Infinite {}
+
+//> INFINITE -> EVALUATE
+impl _Infinite {pub fn evaluate(&self, runtime: &mut Runtime, id: u32, memory: &Vec<Class>) -> Object {
+    //~ INFINITE -> OPERATIONS
+    self.space("Getting infinite value", id);
+    return Object::Infinite(crate::Infinite::new(
```

Clippy (1.91.1)

[warning] unneeded `return` statement

(warning)